test/rules.json

```diff
@@ -3,6 +3,17 @@
   "metadata": {
     "rules_version": "1.3.1"
   },
+  "actions": [
```

Comment:
but if action definitions are now mandatory, is that a breaking change from the perspective of the customers that use custom rules ?

Reply:
Why do you say action definitions are mandatory?

Reply:
Maybe because the docs are a bit confusing about it:
https://github.com/DataDog/libddwaf/blob/1.18.0/UPGRADING.md#upgrading-from-116x-to-117x

> users must now provide action definitions

But, after few paragraphs:

> Finally the following set of default actions are included

So, some of then are added by default, I understand that "actions" are not mandatory. But could it worth to confirm it with libddwaf team.

Reply:
Confirmed, actions is not a mandatory field, it could be undefined. It is mandatory to propagate from rules file or RC, but for this PR, it is not mandatory.